### DISCUSSION TITLE
Delete Pull Request

### DISCUSSION THREAD

**Discussion by Remilekun-Odegbami** [community/community#35901] [Pull Requests] [answered]
How do I delete a pull request in a repo I forked?

**Comment by DenverCoder1** [comment, marked as answer]
You can click the Close Pull Request button to mark the PR as closed. This is your best and simplest option.

If it contains sensitive information, you can contact GitHub support to remove it - https://support.github.com/contact

**Reply by Remilekun-Odegbami** [reply]
I sent a mail to them and I was directed here. I will send another mail. Thank you @DenverCoder1

**Reply by Slashee-the-Cow** [reply]
Is it even still possible to contact them? I can't find a contact link, or create a ticket, or anything, and when I ask their bot to delete it, it says I'm not admin.

I closed the request, I just don't want that PR in there making me look like an idiot.

**Comment by Khushboqof** [comment]
It's helpful to have the delete branch button when a pull request is merged. However, sometimes I'll fork a repository just to fix a small issue and I don't expect to work on it again. I then find after deleting the branch, I have to manually navigate to the repository to delete it. It would be nice if this was offered as an option on the pull request page.

**Comment by RubenJ27** [comment]
Thanks
